main.tf

```diff
@@ -76,3 +76,39 @@ module "ecs_codepipeline" {
   ecs_cluster_name   = "${var.ecs_cluster_name}"
   privileged_mode    = "true"
 }
+
+module "ecs_alarms" {
+  enabled                      = "${var.ecs_alarms_enabled}"
+  source                       = "git::https://github.com/cloudposse/terraform-aws-ecs-cloudwatch-sns-alarms.git?ref=tags/0.2.0"
+  create_sns_topic             = "false"
+  sns_topic_name               = "${var.sns_topic_name}"
+  name                         = "${var.name}"
+  namespace                    = "${var.namespace}"
+  stage                        = "${var.stage}"
+  cluster_name                 = "${var.ecs_cluster_name}"
+  service_name                 = "${module.ecs_alb_service_task.service_name}"
+  cpu_utilization_threshold    = "${var.ecs_alarms_cpu_util_threshold}"
+  memory_utilization_threshold = "${var.ecs_alarms_mem_util_threshold}"
```

Comment:
better to keep consistent names than optimize for length. We didn't optimize for length in `terraform-aws-ecs-cloudwatch-sns-alarms`, so shouldn't do it here.